telemetry/telemetrybuffer.go

```diff
@@ -323,6 +323,16 @@ func (tb *TelemetryBuffer) ConnectToTelemetryService(telemetryNumRetries, teleme
 	}
 }
 
+// ConnectToTelemetry - attempt to connect to telemetry service
+func (tb *TelemetryBuffer) ConnectToTelemetry() {
+	if err := tb.Connect(); err != nil {
+		log.Logf("Connection to telemetry socket failed: %v", err)
```

Comment:
should we use zap logger here?

Reply:
We should but i don't think this package has zaplogger yet. https://github.com/Azure/azure-container-networking/blob/master/telemetry/telemetrybuffer.go#L18